frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -313,6 +313,7 @@ const SECTIONS = updateSectionsWithPlugins({
       {
         key: "enable-public-sharing",
         display_name: t`Enable Public Sharing`,
+        description: t`Enable admins to create publicly viewable links (and embeddable iframes) for Questions and Dashboards.`,
```

Comment:
Update copy [according to the design](https://www.figma.com/file/SAi6pRHuhrWeVRuOHH4evg/Better-communicate-embedding-options-in-Admin-settings?node-id=27%3A996).